outputsheets/d20/fantasy/pdf/common_sheet/block_bio_condensed.xslt

```diff
@@ -50,6 +50,7 @@
 <!-->		<xsl:if test="string-length(translate(normalize-space(concat(description,bio)), ' ', '')) &gt; 0">	-->
 			<fo:page-sequence>
 				<xsl:attribute name="master-reference">Portrait</xsl:attribute>
+				<xsl:attribute name="font-family">NotoSans</xsl:attribute>
```

Comment:
Just a comment, not a fix required, but annotations about what is happening would be handy for the uninitiated. Like "Using notoSans here gives us control over the export fonts used".